packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -41,6 +41,7 @@ export interface AliasConfiguration {
   readonly names: string[],
   readonly acmCertRef: string,
   readonly sslMethod?: SSLMethod,
+  readonly securityPolicy?: SecurityPolicyProtocol
```

Comment:
Can you add documentation here? I know it's not there for the other arguments, but improving the world has to start somewhere. Probably the documentation you put on the type can go here, and the type documentation can be more generic. And please pay attention to documenting the default using `@default`; what does it do if we leave it out?

Reply:
@rix0rrr Updated, put it in the header documentation with the others. Thanks for the feedback!

Reply:
Oh, that's actually not where they're supposed to go. But I'll fix it myself, feels unfair to make you do this chore.